entry/src/main/js/default/pages/index/index.hml

```diff
@@ -0,0 +1,98 @@
+<element src="../../../../../../../smartwash/js/index.hml" name="test"></element>
+<div class="container">
+    <div class="full">
+        <div class="timer">
+            <image class="level_bg" src="/common/images/level_bg.png"></image>
+            <image class="Timmer_bg" src="/common/images/Timer_bg.png"></image>
+            <div class="centered"><text>0:30</text></div>
+            <progress class="min-progress" type="ring" percent= "40" ></progress>
+            <image class="indication_point" src="/common/images/indication_point.png"></image>
+
+            <image src="/common/images/fill.png"class="fill"></image>
+            <image src="/common/images/wash.PNG"class="wash"></image>
+            <image src="/common/images/spin.PNG"class="spin"></image>
+            <image src="/common/images/rinse.PNG"class="rinse"></image>
+            
+            <text class="tlabel1">Fill</text>
+            <text class="tlabel2">Spin</text>
+            <text class="tlabel3">Wash</text>
+            <text class="tlabel4">Rinse</text>
+        </div>
+
+        <div class="dial">
+            <image src="/common/images/band.png" class="band"></image>
+            <image src="/common/images/band_bg.png" class="band_bg"></image>
+            <image src="/common/images/band_grey.png" class="band_grey"></image>
+            <image src="/common/images/selected_bg.png"class="selected_bg"></image>
+            <image src="/common/images/eco wash.png" class="ecowash"></image>
+            <image src="/common/images/wollen.png" class="woll"></image>
+            <image src="/common/images/cotton.png" class="cotton"></image>
+            <image src="/common/images/quick.png" class="quick"></image>
+            <image src="/common/images/rinse.png" class="rinse1"></image>
+            <image src="/common/images/spin.png" class="spin1"></image>
+            <image src="/common/images/mix.png" class="mix"></image>
+            <image src="/common/images/delicates.png" class="delicates"></image>
+            <text class="dlabel1">Delicates</text>
+            <text class="dlabel2">Mix</text>
+            <text class="dlabel3">Cotton</text>
+            <text class="dlabel4">Rinse</text>
+            <text class="dlabel5">Wool</text>
+            <text class="dlabel6">Quick</text>
+            <text class="dlabel7">Spin</text>
+            <text class="dlabel8">Eco Wash</text>
+
+        </div>
+
+        <div class="waterlevel">
```

Comment:
Also maintain setter functions in js file to update data whenever user click happens like for example water level is changed, then setWaterLevel function should take care of updating data.

Also water level data displayed in UI should be defined in js and looped here to display